testcases/OpTestLPM.py

```diff
@@ -52,10 +52,12 @@ def setUp(self):
         self.src_mg_sys = self.cv_HMC.mg_system
         self.dest_mg_sys = self.cv_HMC.tgt_mg_system
         self.oslevel = None
-        self.src_lpar_vios = self.cv_HMC.lpar_vios.split(",")
-        self.dest_lpar_vios = conf.args.remote_lpar_vios.split(",")
         self.slot_num = None
         self.options = None
+        if conf.args.lpar_vios is not None:
```

Comment:
if lpar_vios is not given in conf than.. lpar_vios can have any junk value, but you are checking for None

Reply:
This was defined in OpTestConfiguration.py:

```
hmc = OpTestHMC(self.args.hmc_ip,
                                    self.args.hmc_username,
                                    self.args.hmc_password,
                                    managed_system=self.args.system_name,
                                    tgt_managed_system=self.args.target_system_name,
                                    lpar_name=self.args.lpar_name,
                                    tgt_lpar=self.args.target_lpar_name,
                                    lpar_vios=self.args.lpar_vios,
                                    lpar_prof=self.args.lpar_prof,
                                    lpar_user=self.args.host_user,
                                    lpar_password=self.args.host_password,
                                    logfile=self.logfile
                                    )
```
                                    
Regardless of whether or not lpar_vios is specified in the config file, it is still present in conf.args (with a None value)
So, it seemed kind of redundant to check for: `if 'lpar_vios' in conf.args and conf.args.lpar_vios is not None`

Reply:
ah ok,  than None can be checked by just `if conf.args.lpar_vios`  instead of `conf.args.lpar_vios is not None`